### DISCUSSION TITLE
Please enabled copilot on my account

### DISCUSSION THREAD

**Discussion by kings1990** [community/community#15526] [Copilot Conversations] [answered]
Hello, i have wait for a long time for my apply, could you please pass the apply and enabled it on my account

Thanks very much

![image](https://user-images.githubusercontent.com/8969637/165006512-f7045ffe-6549-42d0-8237-23f0610d5f9b.png)

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
